examples/sombra/main.tf

```diff
@@ -0,0 +1,3 @@
+data "transcend_sombra" "test" {
+  url = "https://test.sombra.trancsend.com/"
```

Comment:
Would add a comment here explaining where this URL is sourced from

Reply:
just added!